padl/transforms.py

```diff
@@ -2248,3 +2301,779 @@ def importdump(transform_or_module):
+                return True
+            output_slice = range(output_slice.stop)[output_slice]
+        if isinstance(input_slice, int) and isinstance(output_slice, list):
+            if input_slice in output_slice:
+                return True
+            return False
+        elif isinstance(input_slice, list) and isinstance(output_slice, int):
+            if output_slice in input_slice:
+                return True
+            return False
+        elif isinstance(input_slice, int) and isinstance(output_slice, int):
+            if input_slice == output_slice:
+                return True
+            return False
+        else:
+            if len(set(input_slice).intersection(set(input_slice))) > 0:
+                return True
+            return False
+
+    def list_all_paths(self, inp_node=None, path=None):
+        """List all paths
+
+        :param inp_node:
+        :param path:
+        """
+        if path is None:
+            path = []
+        if inp_node is None:
+            path = [self.input_node]
+            inp_node = self.input_node
+
+        if len(self.edges[inp_node]) == 0:
+            return path
+
+        return_path = []
+
+        for idx, out_node in enumerate(self.edges[inp_node]):
+            parent_position = self.parents[out_node].index(inp_node)
+
+            output_slice, input_slice = self.edges[inp_node][out_node]
+            if not self._check_edge_compatibility(output_slice, input_slice, parent_position):
+                continue
+            path_copy = path.copy()
+            if isinstance(out_node.transform, Graph):
+                path_copy.append(out_node.transform.input_node)
+                return_path.append(out_node.transform.list_all_paths(out_node.transform.input_node, path_copy))
+            else:
+                path_copy.append(out_node)
+                return_path.append(self.list_all_paths(out_node, path_copy))
+        if len(return_path) == 1:
+            return return_path[0]
+        return return_path
+
+    def _generate_preprocess_dict(self,
+                                  current_node,
+                                  preprocess_edges=None,
+                                  batchify_node=None,
+                                  batchify_found=False):
+        """
+
+        :param current_node:
+        :param preprocess_edges:
+        :param batchify_node:
+        :param batchify_found:
+        :return:
+        """
+        if batchify_node is None:
+            batchify_node = dict()
+        if preprocess_edges is None:
+            preprocess_edges = defaultdict(dict)
+
+        # import pdb;pdb.set_trace()
+        for child in self.edges[current_node]:
+            # import pdb;pdb.set_trace()
+            if child.transform == padl.batch:
+                batchify_found = True
+                if child not in batchify_node:
+                    batchify_node[child] = Node(padl.Identity() - 'batchify Marker')
```

Comment:
The transform `Identity` is defined in this file, so I think you can change `padl.Identity()` to `Identity()`